sass/themes/cr/2019/components/just-in-time-block/_just-in-time-block.scss

```diff
@@ -0,0 +1,6 @@
+.form__row--just-in-time-block {
+  font-family:$montserrat-800;
+  .toggle-link {
+    font-family:$montserrat-800;
+  }
+}

```

Comment:
ooh we need to look into this. When I checked the component it looks like it's still using GT-Walsheim-Bold

<img width="1184" alt="screen shot 2018-12-05 at 15 15 21" src="https://user-images.githubusercontent.com/21171954/49522961-aca8ce80-f8a0-11e8-8899-04a3d5102239.png">